subsys/net/Kconfig.hostname

```diff
@@ -29,9 +29,10 @@ config NET_HOSTNAME_UNIQUE
 	  hostname. For example, zephyr00005e005357 could be the hostname
 	  if this	setting is enabled.
 
-config  NET_DEBUG_HOSTNAME
-	bool "Debug hostname configuration"
-	default n
+config  NET_LOG_LEVEL_HOSTNAME
+	int "Debug level hostname configuration"
+	default 0
```

Comment:
Why we have "default 0" here (and "default 1" in other places). Default level should be ERROR (if not WARNING) consistently, to make sure that by default, errors don't go unnoticed by user.